README.md

```diff
@@ -1,107 +1,77 @@
-did-io
-=================
+# DID Whisper Client _(did-whisper)_
 
 [![Build Status](https://travis-ci.org/digitalbazaar/did-io.png?branch=master)](https://travis-ci.org/digitalbazaar/did-io)
 
-A Decentralized Identifier resolution library for the browser and
-node.js.
+> A [DID](https://w3c-ccg.github.io/did-spec/) (Decentralized Identifier) resolution library for Javascript
```

Comment:
Extra leading ">"?

Reply:
S' how [standard-readme](https://github.com/RichardLitt/standard-readme) does it

Reply:
Ah. Ok.